cnv/cnv_install/upgrading-container-native-virtualization.adoc

```diff
@@ -0,0 +1,19 @@
+[id="upgrading-container-native-virtualization"]
+= Upgrading {CNVProductName}
+include::modules/cnv-document-attributes.adoc[]
+:context: upgrading-container-native-virtualization
+toc::[]
+
+You enable automatic updates during {CNVProductName} xref:../../cnv/cnv_install/installing-container-native-virtualization.adoc#cnv-subscribing-to-hco-catalog_installing-container-native-virtualization[installation].
```

Comment:
I'm uncertain of the phrasing here. How about "Automatic updates are enabled during installation"? or "can be" if it's optional.

Reply:
It does sound strange, but a lot of our content includes "You <verb> <the thing>" phrasing. We try to avoid the passive voice, and "can" is sometimes confusing/unclear.

I am discussing this via email with Steve G and Ryan, because I think we don't *want* it to be optional, but it currently is optional in the UI. I'll leave this comment open and update the phrasing when we come to a conclusion. :)

(Thanks for the review, @stu-gott!)